<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Binary Translator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{

background-color: rgb(74, 74, 74)

}</string>
  </property>
  <widget class="QPushButton" name="TranslateButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>330</y>
     <width>170</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono NL ExtraBold</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(54, 54, 54);
border-radius: 20px;
color: 'lime';
border-style: outset;</string>
   </property>
   <property name="text">
    <string>Translate</string>
   </property>
  </widget>
  <widget class="QLabel" name="Output_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>410</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cascadia Mono</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: 'lime';</string>
   </property>
   <property name="text">
    <string>Output</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
   </property>
  </widget>
  <widget class="QLabel" name="Binary_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>15</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cascadia Mono</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: 'lime'</string>
   </property>
   <property name="text">
    <string>Enter a Binary code</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QPushButton" name="BackButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>330</y>
     <width>170</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono NL ExtraBold</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(54, 54, 54);
border-radius: 20px;
color: 'red';
border-style: outset;</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>400</width>
     <height>250</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cascadia Mono</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: '10px';
background-color: rgb(54, 54, 54);
color: 'lime'</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="OutputBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>430</y>
     <width>400</width>
     <height>150</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cascadia Mono</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: '10px';
background-color: rgb(54, 54, 54);
color: 'lime'</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
